plugins/woocommerce/includes/shortcodes/class-wc-shortcode-order-tracking.php

```diff
@@ -51,7 +51,7 @@ public static function output( $atts ) {
 			} else {
 				$order = wc_get_order( apply_filters( 'woocommerce_shortcode_order_tracking_order_id', $order_id ) );
 
-				if ( $order && $order->get_id() && strtolower( $order->get_billing_email() ) === strtolower( $order_email ) ) {
+				if ( $order && $order->get_id() && 'shop_order_refund' !== $order->get_type() && strtolower( $order->get_billing_email() ) === strtolower( $order_email ) ) {
```

Comment:
There are other order types (other than refunds) that might work well with the tracking shortcake. It'd be best to check that they provide the `get_billing_email()` method instead, for example, by checking that `$order` is an instance of `WC_Order`.
That way the form will happily work with other order types that inherit from `WC_Order` such as subscriptions.